extensions/binding/org.eclipse.smarthome.binding.bosesoundtouch/src/main/java/org/eclipse/smarthome/binding/bosesoundtouch/handler/BoseSoundTouchHandler.java

```diff
@@ -129,8 +129,9 @@ public void updateState(String channelID, State state) {
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
         logger.debug("{}: handleCommand({}, {});", getDeviceName(), channelUID, command);
-        if (getThing().getStatus() != ThingStatus.ONLINE) {
-            openConnection(); // try to reconnect....
+        if (commandExecutor == null) {
+            logger.warn("Can't handle command '{}' for channel '{}' becuase of not initialized connection.", command, channelUID);

```

Comment:
Please reduce logging to `debug` and fix typo `becuase`.

Reply:
Will you also be OK with `info` log?

Reply:
No, see https://www.eclipse.org/smarthome/documentation/development/guidelines.html#e-logging.